patches/tModLoader/Terraria/ModLoader/Engine/ErrorReporting.cs

```diff
@@ -123,4 +123,19 @@ internal static int ShowMessageBoxWithChoices(string title, string message, stri
 		}
 		return buttonID;
 	}
+
+	/// <summary> Various error codes to show in Visual Studio. Mainly used to cross reference with source code. Subject to change if more granular error codes are needed. </summary>
+	internal enum TMLErrorCode
+	{
+		TML001, // tMod file in use
+		TML002, // Any BuildException
+		TML003, // Other exception
```

Comment:
Might make sense to reverse these so we can add more specific ones in a more logical order